tools/scaninc/scaninc.cpp

```diff
@@ -145,6 +145,15 @@ int main(int argc, char **argv)
 
             while ((incDirectiveType = file.ReadUntilIncDirective(path)) != IncDirectiveType::None)
             {
+                if (path.back() == 'h')
+                {
+                    std::string newPath("include/" + path);
+                    if (!CanOpenFile(newPath))
+                    {
+                        newPath = "include/constants/" + path;
```

Comment:
This path assumption should not be hardcoded in the scaninc tool.  Use `includeDirs` to search for the file.

Reply:
I tried that, but I couldn't get it to work correctly.